src/main/resources/webassets/templates/drive.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
```

Comment:
The template files should start with <div class="row">... should not include html, head and body tags